src/MakingSense.AspNetCore.Documentation/DocumentationOptions.cs

```diff
@@ -42,8 +47,13 @@ public class DocumentationOptions
 
 		public TimeSpan CacheMaxAge { get; set; }
 
+		[Obsolete("Use LayoutFilePath property")]
 		public IFileInfo LayoutFile { get; set; }
 
+		public IFileInfo DefaultLayoutFile { get; internal set; }
```

Comment:
Do you think that is it possible to remove these properties (`DefaultNotFoundHtmlFile` and `DefaultLayoutFile`) and convert into variables in `DocumentationMiddleware.ParseLayout()` or fields in `DocumentationMiddleware`?

Reply:
Yes - I think it makes sense to remove both from ´DocumentationOptions`.